src/content/docs/kromer/wallets.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Wallets
+---
+import { Aside } from '@astrojs/starlight/components';
+
+There are several Kromer wallet websites available, anything in this list is considered safe to use.
+
+<Aside type="danger">Please be careful with entering your private key on a wallet not in this list, or while you are getting safety warnings, your private key may be at risk!</Aside>
```

Comment:
More like please be careful with entering your private key on any wallet software but is oki. But regardless I don't quite get the "while you are getting safety warnings"